deckard/http/templates/base.html

```diff
@@ -5,6 +5,7 @@
     <title>Deckard</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="csrf-token" content="{{ csrf_token() }}">
+    <link rel="icon" type="imgage/ico" href="/static/images/deckard.ico">
```

Comment:
type="image/x-icon"

Reply:
`imgage`. I think I copied this from my personal site...what a dummy...

Reply:
updated @paulfalgout